.gitignore

```diff
@@ -23,5 +24,10 @@ coverage
 build/Release
 
 # Dependency directory
-# https://www.npmjs.org/doc/misc/npm-faq.html#should-i-check-my-node_modules-folder-into-git
 node_modules
+
```

Comment:
Hmm. It's still showing up in the PR: https://github.com/enspiral-craftworks/chain-of-positivity/pull/2/files